ext/opcache/ZendAccelerator.c

```diff
@@ -1753,6 +1753,26 @@ zend_op_array *file_cache_compile_file(zend_file_handle *file_handle, int type)
 }
 #endif
 
+int check_persistent_script_access(zend_persistent_script *persistent_script)
+{
+    char *phar_path, *ptr;
+    int ret;
+    if (ZSTR_LEN(persistent_script->script.filename)<8 || memcmp(ZSTR_VAL(persistent_script->script.filename), "phar://", 7))
```

Comment:
Can check `persistent_script->is_phar` here.

Reply:
Nevermind, it looks like `is_phar` refers to directly accessing a .phar file, not phar:// paths.